kubernetes/deployment-staging.tmpl

```diff
@@ -47,6 +47,9 @@ data:
       }
 
       location / {
+        # avoid attacker setting the host value for links in rails_app
+        # https://github.com/rails/rails/issues/29893
+        proxy_set_header X-Forwarded-Host $server_name;
```

Comment:
I believe this will set the `X-Forwarded-Host` to the server block defined in the `server_name panoptes-staging.zooniverse.org;` based on reading at http://nginx.org/en/docs/http/ngx_http_core_module.html#var_server_name

it would be great to double check this and test on staging system before deploying to production.

Reply:
I've manually tested this on the staging api nginx container via 
`curl -v 'https://panoptes-staging.zooniverse.org/oauth/authorize' -H 'Host: www.test.com'`